src/Parser/HolyPriest/CPM_ABILITIES.js

```diff
@@ -26,14 +26,6 @@ const CPM_ABILITIES = [
     getMaxCasts: (cooldown, fightDuration, getAbility, parser) => {
       return calculateMaxCasts(cooldown, fightDuration, 2);
     },
-    recommendedCastEfficiency: 0.50, // This spell is a very odd use, most people use it during movement or for tanks during heavy damage, so this does not really need to be super high but it needs to be high enough that you dont sit at 2 charges
-  },
-  {
```

Comment:
The problem with Desperate Prayer is that one of our traits can cause the cooldown to reset without any indication (as far as I'm aware), so you could end up with something like `7/5 casts`.